tests/test_plotter.py

```diff
@@ -308,7 +308,7 @@ def test_independent_series_merging(self):
         dfr = pd.DataFrame([0, 1, 2, 3], columns=["a"])
         iplot = trappy.ILinePlot(dfr, column=["a"])
         s = {"s1": s1, "s2": s2}
-        merged = iplot._fix_indexes(s)
+        merged = pd.DataFrame(iplot._fix_indexes(s))
```

Comment:
Instead of converting it to a DataFrame to then extract the index and convert it back to a list, I'd rather do this check without pandas, by changing the `assertEquals()` below to:
``` python
self.assertEquals(expected_index, sorted(merged["s1"].keys()))
```